### PR TITLE
Force Stop jobs

### PR DESCRIPTION
**Pull request by EsOsO**
Stop-RSJob hangs waiting the jobs to terminate gracefully, I added a Force switch to the function to kill the job without waiting; useful for long running jobs.

### CODE REVIEW

File: PoshRSJob/PoshRSJob.psm1 [hunk 1]
```diff
@@ -48,7 +48,7 @@ Add-Type -TypeDefinition @"
             public string NewVarName;
         }
     }
-"@ -Language CSharpVersion3
+"@ -Language CSharp
```
Comment (proxb) [inline PoshRSJob/PoshRSJob.psm1]: I thought I had this fixed before but it must have snuck back in during one of my updates. Thanks!

**Comment by proxb** [comment]
Thanks for submitting a PR on this! I'll do some quick testing and if it looks OK, I'll move forward and merge this.

**Comment by EsOsO** [comment]
if can help, after forcing the stop, I can no longer remove the job:
```powershell
Remove-RSJob : Unable to remove job a593ad54-8ae2-41e5-9c56-4302522ff689
At line:1 char:13
+ Get-RSJob | Remove-RSJob
+             ~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Remove-RSJob
```

**Comment by proxb** [comment]
That is interesting. Will have to see why that is.

**Comment by proxb** [comment]
I fixed the issue with Remove-RSJob.

**Comment by EsOsO** [comment]
Great, I'll take a look.

**Pull request merged**